internal/pkg/composable/providers/kubernetessecrets/config.go

```diff
@@ -10,4 +10,10 @@ import "github.com/elastic/elastic-agent-autodiscover/kubernetes"
 type Config struct {
 	KubeConfig        string                       `config:"kube_config"`
 	KubeClientOptions kubernetes.KubeClientOptions `config:"kube_client_options"`
+
+	TTL int `config:"ttl"`
```

Comment:
You wrap this value with `time.Duration` when it is being used. Why not just use `time.Duration` in the configuration? That would allow the TTL to be set using `10s` instead of it being set to a specific unit.

Reply:
> Why not just use time.Duration in the configuration? 

Like it is now as a string? Or use time duration exactly in this place?

Reply:
I left it as a string, because I don't see the place to set it to the default in case the user sets it wrong :thinking: